ai-machine-learning/thoth-s2i-advise/parameter.md

```diff
@@ -0,0 +1,27 @@
+Now finally let's explore the options thoth s2i build image offers - 
+
+Thoth’s s2i container images can be configured using environment variables supplied to the build config:
+
+`THOTH_ADVISE` - always use the recommended stack by Thoth (even if Pipfile.lock is present in the repo)
+
+`THOTH_ASSEMBLE_DEBUG` - run s2i’s assemble script in verbose mode
+
+`THOTH_DRY_RUN` - submit stack to Thoth’s recommendation engine but do not use the recommended Pipfile.lock file, use the Pipfile.lock file present in the repo instead
+
+`THOTH_FROM_MASTER` - Use Thamos from git instead of a PyPI release - handy if the released Thamos has a bug which was fixed in the master branch
+
+`THOTH_HOST` - Thoth’s host to reach out to for recommendations (defaults to prod deployment at khemenu.thoth-station.ninja)
+
+`THOTH_ERROR_FALLBACK` - fallback to the Pipfile.lock present in the repository if the submitted Thoth analysis fails
+
```

Comment:
`THOTH_PROVENANCE_CHECK` (should be always 0 in this scenario?)

Reply:
I agree! Will put it in the branch also.

Reply:
Should we mention it here?